lib/init.g

```diff
@@ -560,6 +560,9 @@ BindGlobal( "ShowKernelInformation", function()
   if GAPInfo.KernelInfo.KernelDebug then
     Add( config, "KernelDebug" );
   fi;
+  if GAPInfo.KernelInfo.MemCheck then
+    Add(config, "MemCheck (EXPERIMENTAL)");
```

Comment:
Experimental might be a bad word. If we later changed GCs to boehm it would get removed, and I could imagine there could be later changes which significantly change the behaviour. Also I don't really want people to start using this as their standard GAP, because it is slower and more limited in a number of ways (doesn't give memory back to the OS for example).